code/game/machinery/traitorous_cloner.dm

```diff
@@ -0,0 +1,142 @@
+//Experimental cloner; clones a body regardless of the owner's status, letting a ghost control it instead
+
+/obj/machinery/clonepod/experimental/traitorous
+	name = "cloning pod"
+	desc = "An ancient cloning pod. It seems to be an early prototype of the experimental cloners used in Nanotrasen Stations."
+	icon = 'icons/obj/machines/cloning.dmi'
+	icon_state = "pod_0"
+	req_access = null
+	circuit = /obj/item/circuitboard/machine/clonepod/experimental/traitorous
+	internal_radio = FALSE
+	var/emmaged = FALSE
+
+//Start growing a human clone in the pod!
+/obj/machinery/clonepod/experimental/traitorous/growclone(clonename, ui, mutation_index, mindref, last_death, datum/species/mrace, list/features, factions, list/quirks, datum/bank_account/insurance)
+	if(panel_open)
+		return NONE
+	if(mess || attempting)
+		return NONE
+
+	attempting = TRUE //One at a time!!
+	countdown.start()
+
+	var/mob/living/carbon/human/H = new /mob/living/carbon/human(src)
+
+	H.hardset_dna(ui, mutation_index, H.real_name, null, mrace, features)
+
+	if(efficiency > 2)
+		var/list/unclean_mutations = (GLOB.not_good_mutations|GLOB.bad_mutations)
```

Comment:
This should be defined only once, somewhere else, perhaps as a static var on this type

Reply:
Don't understand what you mean

Reply:
I am pretty sure that code works though, cuz I did get acid flesh after cloning once.

Reply:
I think you're overengineering this, this is original code I didn't touch it, if you try to define this as a static var when the file is being compiled, you will get both of these globals as null, this is just leading to useless fuckery.

Reply:
Also, this is a bit-wise operation, it's one of the cheapest ones on the cpu, it's speed is hilarious, so this shouldn't be a concern at all.

Reply:
No it's not a bitwise operation you utter bafoon it's on TWO LISTS

Reply:
In fact this is one of the slowest possible ways you could combine to lists. God. It literally allocates for an entire new list and combines the two lists, all while making sure there are no duplicates.

Reply:
Listen to the god damn review comments and make it static. We know what were doing. my god